webpack.config.js

```diff
@@ -17,13 +16,14 @@ const ExtraWatchWebpackPlugin = require("extra-watch-webpack-plugin");
 
 // Paths
 const npmPackage = 'node_modules/';
-const srcDir = process.env.npm_package_srcDir;
-const outputDir = process.env.npm_package_distDir;
-const kssSrcDir = process.env.npm_package_kssSrcDir;
+const srcDir = path.resolve( __dirname, process.env.npm_package_srcDir );
```

Comment:
i think it's safest to leave the srcDir as a relative path. If it's absolute, I think there's a risk of absolute paths ending up in our generated assets, e.g. references to background images that are too big to be inlined as data.

Reply:
Nevermind.